src/render/tutorials/configurable-api-gateway/index.html.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: Configurable API gateway
-author: marcinkp
+author: mateuszgrab-cognifide
```

Comment:
The same. Maybe we can specify here more authors?

Reply:
I haven't managed to find a way to specify two authors, I've recovered original author.